src/search/tasks/negated_axioms_task.h

```diff
@@ -0,0 +1,62 @@
+#ifndef TASKS_NEGATED_AXIOMS_TASK_H
+#define TASKS_NEGATED_AXIOMS_TASK_H
+
+#include "delegating_task.h"
+
+#include "../task_proxy.h" // TODO: can we get rid of this include?
+
+#include "../algorithms/sccs.h"
+
+#include <set>
+
+namespace plugins {
+class Options;
+}
+
+namespace tasks {
+struct NegatedAxiom {
+    FactPair head;
+    std::vector<FactPair> condition;
+
+    // TODO: clang suggested moving the second argument, is this correct? (clang doesn't complain at least :))
```

Comment:
I did not see anything that would render this wrong.

Reply:
Removed the comment, thanks!